rocky/rocky/views/ooi_report.py

```diff
@@ -103,7 +103,7 @@ def build_finding_dict(
     finding_dict["finding_type"] = finding_type_dict
 
     if finding_dict["description"] is None:
-        finding_dict["description"] = finding_type_dict["description"]
+        finding_dict["description"] = finding_type_dict["description"] if "description" in finding_type_dict else ""
```

Comment:
Woudnt using a get('description', '') with a default empty string be more clean?